src/Http/Requests/News/NewsRequest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests\News;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Mews\Purifier\Facades\Purifier;
+
+class NewsRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "title" => "required|string",
+            "text" => "required|string",
+        ];
+    }
+
+    protected function prepareForValidation(): void
+    {
+        $cleanText = Purifier::clean($this->text);
```

Comment:
Please explain here what this purifier do.

Reply:
Purifier cleans from html scripts and leaves only md but create html <p> tag and I haven't seen in documentation to change it

Reply:
md

Reply:
i mean clear md can't include js and html